ui/pages/token-allowance/token-allowance.test.js

```diff
@@ -245,4 +253,28 @@ describe('TokenAllowancePage', () => {
     fireEvent.click(gotIt);
     expect(gotIt).not.toBeInTheDocument();
   });
+
+  it('should show hardware wallet info text', () => {
+    const { queryByText, getByText, getByTestId } = renderWithProvider(
+      <TokenAllowance {...props} />,
+      store,
+    );
+
+    const textField = getByTestId('custom-spending-cap-input');
+    fireEvent.change(textField, { target: { value: '1' } });
+
+    const nextButton = getByText('Next');
```

Comment:
nice! tests look good. could be handy to add an additional null check to check the text is not showing on the first page 

```suggestion
    expect(queryByText('Prior to clicking confirm:')).toBeNull();

    const nextButton = getByText('Next');
```

Reply:
Good suggestion! Thanks!
Changed it here: 455663deac8390e9a234fbf8e4bbcf389508066c